src/osx/slider_osx.cpp

```diff
@@ -346,33 +383,25 @@ wxSize wxSlider::DoGetBestSize() const
         wxString text;
 
         // Get maximum text label width and height
-        text.Printf( wxT("%d"), ValueInvertOrNot( m_rangeMin ) );
+        text.Printf( "%d", ValueInvertOrNot( m_rangeMin ) );
         GetTextExtent(text, &mintwidth, &mintheight);
-        text.Printf( wxT("%d"), ValueInvertOrNot( m_rangeMax ) );
+        text.Printf( "%d", ValueInvertOrNot( m_rangeMax ) );
         GetTextExtent(text, &maxtwidth, &maxtheight);
 
-        if (maxtheight > mintheight)
-            textheight = maxtheight;
-        else
-            textheight = mintheight;
-
-        if (maxtwidth > mintwidth)
-            textwidth = maxtwidth;
-        else
-            textwidth = mintwidth;
+        textheight = MAX(mintheight,maxtheight);
+        textwidth  = MAX(mintwidth, maxtwidth);
```

Comment:
I would like to do it but the source code had to be basically be re-written. The source code was old inappropriate Carbon code (see my comments below). A lot had to be changed to make it compatible with Cocoa (with respect to alignment and outlook).

PS: There is some more old code inside that can be removed (it is not necessary to change it as it still works but makes the code more complicated than it actually is). I did not remove it because re-writing everything would have been easier. And I already feared that some comments would pop up because of all these changes.